galasa-maven-plugin/pom.xml

```diff
@@ -78,107 +78,231 @@
 		<maven.build.timestamp.format>yyyyMMddHHmm</maven.build.timestamp.format>
 	</properties>
 
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>dev.galasa</groupId>
+				<artifactId>dev.galasa.plugin.common</artifactId>
+				<version>0.32.0</version>
+			</dependency>
+
+			<dependency>
+				<groupId>dev.galasa</groupId>
+				<artifactId>dev.galasa.plugin.common.impl</artifactId>
+				<version>0.32.0</version>
+			</dependency>
+
+			<dependency>
+				<groupId>dev.galasa</groupId>
+				<artifactId>dev.galasa.plugin.common.test</artifactId>
+				<version>0.32.0</version>
+				<scope>test</scope>
+			</dependency>

```

Comment:
Versions need adjusting

Reply:
done.

Reply:
fixed.

Reply:
Can't mark this conversation as resolved.